semgrep/semgrep/metric_manager.py

```diff
@@ -38,6 +40,9 @@ def __init__(self) -> None:
         self._run_time: Optional[float] = None
         self._total_bytes_scanned: Optional[int] = None
         self._errors: List[str] = []
+        self._rule_hashes: List[str] = []
```

Comment:
What's the difference between _rules_hash and rule_hashes? (Specifically, what's the former for?)